crates/core/src/contracts/tests/account.rs

```diff
@@ -0,0 +1,32 @@
+use rstest::*;
+use starknet::core::types::{BlockId, BlockTag};
+
+use crate::client::api::KakarotStarknetApi;
+use crate::contracts::account::{Account, KakarotAccount};
+use crate::test_utils::deploy_helpers::{get_contract, get_contract_deployed_bytecode, KakarotTestEnvironmentContext};
+use crate::test_utils::fixtures::kakarot_test_env_ctx;
+
+#[rstest]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_bytecode(kakarot_test_env_ctx: KakarotTestEnvironmentContext) {
+    // Given
+    let contract_name = "Counter";
+    let (client, _kakarot, _counter, counter_eth_address) = kakarot_test_env_ctx.resources_with_contract(contract_name);
+
+    let contract = get_contract(contract_name);
+    let expected_bytecode = get_contract_deployed_bytecode(contract);
+
+    let starknet_block_id = BlockId::Tag(BlockTag::Latest);
+    let counter_starknet_address = client
```

Comment:
If you use the above `_counter`, you can access the field `addresses` which then gives you the `starknet_address`.

Reply:
Ohh nice let me use this 🫡